ktor-server/ktor-server-plugins/ktor-server-cors/jvmAndNix/src/io/ktor/server/plugins/cors/CORS.kt

```diff
@@ -251,12 +255,13 @@ private fun corsCheckOrigins(
     allowsAnyHost: Boolean,
     hostsNormalized: Set<String>,
     hostsWithWildcard: Set<Pair<String, String>>,
+    originPredicates: List<(String) -> Boolean>,
     numberRegex: Regex
 ): Boolean {
     val normalizedOrigin = normalizeOrigin(origin, numberRegex)
     return allowsAnyHost || normalizedOrigin in hostsNormalized || hostsWithWildcard.any { (prefix, suffix) ->
         normalizedOrigin.startsWith(prefix) && normalizedOrigin.endsWith(suffix)
-    }
+    } || originPredicates.any { it(origin) }
```

Comment:
checkstyle fails here. `}` should be on a new line

Reply:
Unless I'm mistaken, that's not what the build says https://ktor.teamcity.com/viewLog.html?buildId=150645&tab=Inspection&buildTypeId=Ktor_KtorCodeStyleVerifyKtLint

I fixed the "issue" reported there. I also strongly disagree with code style that reduces readability, but whatever. It should pass now.